### PR TITLE
Eiffelstudio patch

### PR DESCRIPTION
**Pull request by sstillwell**
- [y] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [y] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [y] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [y] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
This should resolve an outstanding issue with eiffelstudio and still allow freeciv to work properly.  First PR, so please forgive and help me correct any problems with process.

### CODE REVIEW

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by manuseiffel** [comment]
Looks like the commit to bump gtk revision is causing the build to fail. Is this expected?

File: Formula/gtk+.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Gtkx < Formula
   stable do
     url "https://download.gnome.org/sources/gtk+/2.24/gtk+-2.24.31.tar.xz"
     sha256 "68c1922732c7efc08df4656a5366dcc3afdc8791513400dac276009b40954658"
+    revision 1
```
Comment (dunn) [inline Formula/gtk+.rb]: Oh, this needs to be outside the stable block, below `homepage` above.

**Comment by manuseiffel** [comment]
@nijikon Do you mind fixing the revision location? Thanks!

**Comment by sstillwell** [comment]
Fixed.

**Comment by dunn** [comment]
Merged as b5f97590f038be815fb4b9cb5f654b14099ab776, thanks!

**Pull request closed without merging**